WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -164,7 +176,11 @@ private extension PointOfSaleCollectCashView {
     }
 
     private func conditionalPadding(_ padding: CGFloat) -> CGFloat {
-        dynamicTypeSize.isAccessibilitySize ? 0 : padding
+        if keyboardFrame.intersects(buttonFrame) {
+            return 4
```

Comment:
nit: can make this a constant, for easier lookup when consolidating the spacing and padding in https://github.com/woocommerce/woocommerce-ios/issues/15111.